cmd/conf/database.go

```diff
@@ -60,6 +63,19 @@ func (c *PersistentConfig) ResolveDirectoryNames() error {
 		return fmt.Errorf("database in --persistent.chain (%s) directory, try specifying parent directory", c.Chain)
 	}
 
+	// Make Log directory relative to persistent storage directory if not already absolute
+	if !filepath.IsAbs(c.LogDir) {
+		c.LogDir = path.Join(c.Chain, c.LogDir)
+	}
+	if c.LogDir != c.Chain {
+		err = os.MkdirAll(c.LogDir, os.ModePerm)
```

Comment:
is os.ModePerm too permissive? It's 0777 which means anyone can read/write/list contents https://pkg.go.dev/os#pkg-constants

Reply:
I had initially set it because Chain and GlobalConfig had ModePerm too. But seems likes 744 is a better option

Reply:
755 is probably what you want since, for directories, the bit at position zero for each octal digit allows the directory to be listed

Reply:
turns out there's a reason we're setting it to 0777 https://docs.arbitrum.io/node-running/how-tos/running-a-full-node#note-on-permissions